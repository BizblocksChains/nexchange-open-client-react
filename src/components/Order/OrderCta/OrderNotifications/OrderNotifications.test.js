import React from 'react';
import { shallow } from 'enzyme';
import OrderNotifications from './OrderNotifications';

describe('OrderNotifications', () => {
  it('renders correctly', () => {
    const wrapShallow = shallow(<OrderNotifications email="" message={{ text: 'Hello', error: false }} />).dive();
    expect(wrapShallow).toMatchSnapshot();
  });
});
