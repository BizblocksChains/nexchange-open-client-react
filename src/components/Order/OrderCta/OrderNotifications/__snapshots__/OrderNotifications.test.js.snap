// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderNotifications renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <I18n
    ns="translations"
>
    [Function]
</I18n>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="box container"
>
        <div
                className="row"
        >
                <div
                        className="col-xs-12 visible-xs text-center"
                >
                        <img
                                alt="notify.alt"
                                className="img"
                                src="get-notified.svg"
                        />
                </div>
                <div
                        className="col-xs-12 col-sm-7 text"
                >
                        <h2
                                className="title"
                        >
                                notify.explanation
                        </h2>
                        <form
                                onSubmit={undefined}
                        >
                                <div
                                        className="form-group"
                                >
                                        <input
                                                disabled={undefined}
                                                name="email"
                                                onChange={undefined}
                                                placeholder="notify.email"
                                                required={true}
                                                type="email"
                                                value=""
                                        />
                                        <button
                                                className="btn btn-primary btn"
                                                disabled={undefined}
                                                type="submit"
                                        >
                                                generalterms.submit
                                        </button>
                                        <h4
                                                className="message"
                                        >
                                                Hello
                                        </h4>
                                </div>
                        </form>
                </div>
                <div
                        className="col-ms-2 col-sm-5 hidden-xs text-center"
                >
                        <img
                                alt="notify.alt"
                                className="img"
                                src="get-notified.svg"
                        />
                </div>
        </div>
</div>,
      "className": "col-xs-12",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="row"
>
          <div
                    className="col-xs-12 visible-xs text-center"
          >
                    <img
                              alt="notify.alt"
                              className="img"
                              src="get-notified.svg"
                    />
          </div>
          <div
                    className="col-xs-12 col-sm-7 text"
          >
                    <h2
                              className="title"
                    >
                              notify.explanation
                    </h2>
                    <form
                              onSubmit={undefined}
                    >
                              <div
                                        className="form-group"
                              >
                                        <input
                                                  disabled={undefined}
                                                  name="email"
                                                  onChange={undefined}
                                                  placeholder="notify.email"
                                                  required={true}
                                                  type="email"
                                                  value=""
                                        />
                                        <button
                                                  className="btn btn-primary btn"
                                                  disabled={undefined}
                                                  type="submit"
                                        >
                                                  generalterms.submit
                                        </button>
                                        <h4
                                                  className="message"
                                        >
                                                  Hello
                                        </h4>
                              </div>
                    </form>
          </div>
          <div
                    className="col-ms-2 col-sm-5 hidden-xs text-center"
          >
                    <img
                              alt="notify.alt"
                              className="img"
                              src="get-notified.svg"
                    />
          </div>
</div>,
        "className": "box container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="col-xs-12 visible-xs text-center"
>
              <img
                            alt="notify.alt"
                            className="img"
                            src="get-notified.svg"
              />
</div>,
            <div
              className="col-xs-12 col-sm-7 text"
>
              <h2
                            className="title"
              >
                            notify.explanation
              </h2>
              <form
                            onSubmit={undefined}
              >
                            <div
                                          className="form-group"
                            >
                                          <input
                                                        disabled={undefined}
                                                        name="email"
                                                        onChange={undefined}
                                                        placeholder="notify.email"
                                                        required={true}
                                                        type="email"
                                                        value=""
                                          />
                                          <button
                                                        className="btn btn-primary btn"
                                                        disabled={undefined}
                                                        type="submit"
                                          >
                                                        generalterms.submit
                                          </button>
                                          <h4
                                                        className="message"
                                          >
                                                        Hello
                                          </h4>
                            </div>
              </form>
</div>,
            <div
              className="col-ms-2 col-sm-5 hidden-xs text-center"
>
              <img
                            alt="notify.alt"
                            className="img"
                            src="get-notified.svg"
              />
</div>,
          ],
          "className": "row",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="notify.alt"
                className="img"
                src="get-notified.svg"
/>,
              "className": "col-xs-12 visible-xs text-center",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "notify.alt",
                "className": "img",
                "src": "get-notified.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2
                  className="title"
>
                  notify.explanation
</h2>,
                <form
                  onSubmit={undefined}
>
                  <div
                                    className="form-group"
                  >
                                    <input
                                                      disabled={undefined}
                                                      name="email"
                                                      onChange={undefined}
                                                      placeholder="notify.email"
                                                      required={true}
                                                      type="email"
                                                      value=""
                                    />
                                    <button
                                                      className="btn btn-primary btn"
                                                      disabled={undefined}
                                                      type="submit"
                                    >
                                                      generalterms.submit
                                    </button>
                                    <h4
                                                      className="message"
                                    >
                                                      Hello
                                    </h4>
                  </div>
</form>,
              ],
              "className": "col-xs-12 col-sm-7 text",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "notify.explanation",
                  "className": "title",
                },
                "ref": null,
                "rendered": "notify.explanation",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <div
                    className="form-group"
>
                    <input
                                        disabled={undefined}
                                        name="email"
                                        onChange={undefined}
                                        placeholder="notify.email"
                                        required={true}
                                        type="email"
                                        value=""
                    />
                    <button
                                        className="btn btn-primary btn"
                                        disabled={undefined}
                                        type="submit"
                    >
                                        generalterms.submit
                    </button>
                    <h4
                                        className="message"
                    >
                                        Hello
                    </h4>
</div>,
                  "onSubmit": undefined,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <input
                        disabled={undefined}
                        name="email"
                        onChange={undefined}
                        placeholder="notify.email"
                        required={true}
                        type="email"
                        value=""
/>,
                      <button
                        className="btn btn-primary btn"
                        disabled={undefined}
                        type="submit"
>
                        generalterms.submit
</button>,
                      <h4
                        className="message"
>
                        Hello
</h4>,
                    ],
                    "className": "form-group",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "disabled": undefined,
                        "name": "email",
                        "onChange": undefined,
                        "placeholder": "notify.email",
                        "required": true,
                        "type": "email",
                        "value": "",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "input",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "generalterms.submit",
                        "className": "btn btn-primary btn",
                        "disabled": undefined,
                        "type": "submit",
                      },
                      "ref": null,
                      "rendered": "generalterms.submit",
                      "type": "button",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Hello",
                        "className": "message",
                      },
                      "ref": null,
                      "rendered": "Hello",
                      "type": "h4",
                    },
                  ],
                  "type": "div",
                },
                "type": "form",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="notify.alt"
                className="img"
                src="get-notified.svg"
/>,
              "className": "col-ms-2 col-sm-5 hidden-xs text-center",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "notify.alt",
                "className": "img",
                "src": "get-notified.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="box container"
>
          <div
                    className="row"
          >
                    <div
                              className="col-xs-12 visible-xs text-center"
                    >
                              <img
                                        alt="notify.alt"
                                        className="img"
                                        src="get-notified.svg"
                              />
                    </div>
                    <div
                              className="col-xs-12 col-sm-7 text"
                    >
                              <h2
                                        className="title"
                              >
                                        notify.explanation
                              </h2>
                              <form
                                        onSubmit={undefined}
                              >
                                        <div
                                                  className="form-group"
                                        >
                                                  <input
                                                            disabled={undefined}
                                                            name="email"
                                                            onChange={undefined}
                                                            placeholder="notify.email"
                                                            required={true}
                                                            type="email"
                                                            value=""
                                                  />
                                                  <button
                                                            className="btn btn-primary btn"
                                                            disabled={undefined}
                                                            type="submit"
                                                  >
                                                            generalterms.submit
                                                  </button>
                                                  <h4
                                                            className="message"
                                                  >
                                                            Hello
                                                  </h4>
                                        </div>
                              </form>
                    </div>
                    <div
                              className="col-ms-2 col-sm-5 hidden-xs text-center"
                    >
                              <img
                                        alt="notify.alt"
                                        className="img"
                                        src="get-notified.svg"
                              />
                    </div>
          </div>
</div>,
        "className": "col-xs-12",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="row"
>
            <div
                        className="col-xs-12 visible-xs text-center"
            >
                        <img
                                    alt="notify.alt"
                                    className="img"
                                    src="get-notified.svg"
                        />
            </div>
            <div
                        className="col-xs-12 col-sm-7 text"
            >
                        <h2
                                    className="title"
                        >
                                    notify.explanation
                        </h2>
                        <form
                                    onSubmit={undefined}
                        >
                                    <div
                                                className="form-group"
                                    >
                                                <input
                                                            disabled={undefined}
                                                            name="email"
                                                            onChange={undefined}
                                                            placeholder="notify.email"
                                                            required={true}
                                                            type="email"
                                                            value=""
                                                />
                                                <button
                                                            className="btn btn-primary btn"
                                                            disabled={undefined}
                                                            type="submit"
                                                >
                                                            generalterms.submit
                                                </button>
                                                <h4
                                                            className="message"
                                                >
                                                            Hello
                                                </h4>
                                    </div>
                        </form>
            </div>
            <div
                        className="col-ms-2 col-sm-5 hidden-xs text-center"
            >
                        <img
                                    alt="notify.alt"
                                    className="img"
                                    src="get-notified.svg"
                        />
            </div>
</div>,
          "className": "box container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="col-xs-12 visible-xs text-center"
>
                <img
                                alt="notify.alt"
                                className="img"
                                src="get-notified.svg"
                />
</div>,
              <div
                className="col-xs-12 col-sm-7 text"
>
                <h2
                                className="title"
                >
                                notify.explanation
                </h2>
                <form
                                onSubmit={undefined}
                >
                                <div
                                                className="form-group"
                                >
                                                <input
                                                                disabled={undefined}
                                                                name="email"
                                                                onChange={undefined}
                                                                placeholder="notify.email"
                                                                required={true}
                                                                type="email"
                                                                value=""
                                                />
                                                <button
                                                                className="btn btn-primary btn"
                                                                disabled={undefined}
                                                                type="submit"
                                                >
                                                                generalterms.submit
                                                </button>
                                                <h4
                                                                className="message"
                                                >
                                                                Hello
                                                </h4>
                                </div>
                </form>
</div>,
              <div
                className="col-ms-2 col-sm-5 hidden-xs text-center"
>
                <img
                                alt="notify.alt"
                                className="img"
                                src="get-notified.svg"
                />
</div>,
            ],
            "className": "row",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="notify.alt"
                  className="img"
                  src="get-notified.svg"
/>,
                "className": "col-xs-12 visible-xs text-center",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "notify.alt",
                  "className": "img",
                  "src": "get-notified.svg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2
                    className="title"
>
                    notify.explanation
</h2>,
                  <form
                    onSubmit={undefined}
>
                    <div
                                        className="form-group"
                    >
                                        <input
                                                            disabled={undefined}
                                                            name="email"
                                                            onChange={undefined}
                                                            placeholder="notify.email"
                                                            required={true}
                                                            type="email"
                                                            value=""
                                        />
                                        <button
                                                            className="btn btn-primary btn"
                                                            disabled={undefined}
                                                            type="submit"
                                        >
                                                            generalterms.submit
                                        </button>
                                        <h4
                                                            className="message"
                                        >
                                                            Hello
                                        </h4>
                    </div>
</form>,
                ],
                "className": "col-xs-12 col-sm-7 text",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "notify.explanation",
                    "className": "title",
                  },
                  "ref": null,
                  "rendered": "notify.explanation",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <div
                      className="form-group"
>
                      <input
                                            disabled={undefined}
                                            name="email"
                                            onChange={undefined}
                                            placeholder="notify.email"
                                            required={true}
                                            type="email"
                                            value=""
                      />
                      <button
                                            className="btn btn-primary btn"
                                            disabled={undefined}
                                            type="submit"
                      >
                                            generalterms.submit
                      </button>
                      <h4
                                            className="message"
                      >
                                            Hello
                      </h4>
</div>,
                    "onSubmit": undefined,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        <input
                          disabled={undefined}
                          name="email"
                          onChange={undefined}
                          placeholder="notify.email"
                          required={true}
                          type="email"
                          value=""
/>,
                        <button
                          className="btn btn-primary btn"
                          disabled={undefined}
                          type="submit"
>
                          generalterms.submit
</button>,
                        <h4
                          className="message"
>
                          Hello
</h4>,
                      ],
                      "className": "form-group",
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "disabled": undefined,
                          "name": "email",
                          "onChange": undefined,
                          "placeholder": "notify.email",
                          "required": true,
                          "type": "email",
                          "value": "",
                        },
                        "ref": null,
                        "rendered": null,
                        "type": "input",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "generalterms.submit",
                          "className": "btn btn-primary btn",
                          "disabled": undefined,
                          "type": "submit",
                        },
                        "ref": null,
                        "rendered": "generalterms.submit",
                        "type": "button",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "Hello",
                          "className": "message",
                        },
                        "ref": null,
                        "rendered": "Hello",
                        "type": "h4",
                      },
                    ],
                    "type": "div",
                  },
                  "type": "form",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="notify.alt"
                  className="img"
                  src="get-notified.svg"
/>,
                "className": "col-ms-2 col-sm-5 hidden-xs text-center",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "notify.alt",
                  "className": "img",
                  "src": "get-notified.svg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
